ngraph_bridge/ie_executable.cc

```diff
@@ -63,14 +100,32 @@ IE_Executable::IE_Executable(shared_ptr<Function> func, string device)
 bool IE_Executable::call(const vector<shared_ptr<runtime::Tensor>>& outputs,
                          const vector<shared_ptr<runtime::Tensor>>& inputs) {
   InferenceEngine::InputsDataMap input_info = m_network.getInputsInfo();
-  if (input_info.size() != inputs.size()) {
+  if (input_info.size() != (inputs.size() + m_hoisted_params.size())) {
     THROW_IE_EXCEPTION
         << "Function inputs number differ from number of given inputs";
   }
 
+  //  Prepare input blobs
   size_t i = 0;
   for (const auto& it : input_info) {
-    shared_ptr<IETensor> tv = static_pointer_cast<IETensor>(inputs[i]);
+    shared_ptr<IETensor> tv;
+    // First check if there were any constants we converted to parameters
+    if (m_hoisted_params.size() > 0) {
+      // We only support one parameter replacement for nullary functions
+      CHECK(m_hoisted_params.size() == 1)
+          << "Multiple input constants were converted to parameters.";
+      CHECK(input_info.size() == 1) << "Expecting one input that was converted "
+                                       "from constant to parameter.";
+      auto input = m_hoisted_params.find(it.first);
+      if (input != m_hoisted_params.end()) {
```

Comment:
This check means we were able to convert 1 const to param but the input value does not exist?

Reply:
Yes.

Reply:
Can that happen?

Reply:
Unlikely, but it could happen. The parameter to look up here is coming from IE.